tuxemon/technique.py

```diff
@@ -366,7 +412,6 @@ def apply_lifeleech(self, user: Monster, target: Monster) -> EffectResult:
         }
 
     def poison(self, target: Monster) -> EffectResult:
-        assert self.link
```

Comment:
assert statements are useful for debugging, or catching errors at runtime.  im not familiar with this one, but why was it removed?

Reply:
was causing crashes